Automattic/jetpack
# [Boost] Add new row types to the Premium Image Guide

Fixes #31428 and #31436

Add support for showing rows which failed to load (rather thna breaking the ui) _and_ show proper rows for missing images.

## Proposed changes:
* Refactor the way rows are handled
* Add "broken" rows for things that fail to load
* Add support for showing broken images on the site -- not a broken bit of the plugin, detecting a new problem type. :) 

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


no

## Testing instructions:
* Add broken image links to your site
* Run the image guide, make sure it shows images
* Add an invalid image report issue to your database (change its type to something invalid), verify the ui doesn't break as a result.
